File: clients/typescript-angular/API/Client/ApiKey.ts
/// <reference path="api.d.ts" />

namespace API.Client {
    'use strict';

    export interface ApiKey {

        id: string;

        secret: string;

        name: string;

        nonce: number;

        cidr?: string;

        permissions?: Array<string>;

        enabled?: boolean;

        userId: number;

        created?: date;
    }

}

File: clients/scala/src/main/scala/io/swagger/client/model/ApiKey.scala
package io.swagger.client.model

import java.util.Date



case class ApiKey (
  id: String,
  secret: String,
  name: String,
  nonce: Number,
  cidr: String,
  permissions: List[String],
  enabled: Boolean,
  userId: Number,
  created: Date)
  
